client/react/containers/Signup/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
```

Comment:
Aren't we making all containers regular react components? Or is it just the ones we have to?

Reply:
Just so everyone else is aware, Jugal and I discussed this and we are happy with the convention that every container is a stateless component unless it needs to do something with the lifecycle methods in React.Component.

Is everyone else cool with that?